### ISSUE TITLE
Block categories

### ISSUE THREAD

**Issue by ghost** [microsoftsam/pxt-holidays#6]
How can I show block categories? Most default pxt have block categories, big buttons you click on to see more specific blocks .

**Comment by samelhusseini** [comment]
Yes, this sample doesn't use categories, but here's how it's done:
remove ``hasCategories: false`` from pxtarget.json
https://github.com/samelhusseini/pxt-holidays/blob/master/pxtarget.json#L54

remove ``categories: false`` from main.tsx
https://github.com/samelhusseini/pxt-holidays/blob/master/site/src/main.tsx#L212

And for the purposes of this simple demo, I've removed a number of built in categories, you'll want to enable those perhaps here: 
https://github.com/samelhusseini/pxt-holidays/blob/master/pxtarget.json#L14
